bot/cogs/grades_cog/grades_cog.py

```diff
@@ -50,20 +53,69 @@ def load_data(self):
         """
     )
     @ext.short_help('Attempts to give more information about courses at Clemson University')
-    @ext.example(('grades cpsc-1010', 'grades cpsc-1010 2017'))
-    async def grades(self, ctx, course: str, year: int=MIN_YEAR):
+    @ext.example(
+        ('grades math-2060', 'grades math-2060 2017', 'grades math-2060 honors', 'grades math-2060 2017 honors'))
+    async def grades(self, ctx, course: str, *args):
```

Comment:
we should default to all for this command unless other wise stated, use the referenced d.py converters we discussed, if we want to allow for either honors OR nonhonors specifying let me know and ill show you how to write a custom converter